src/main/java/UC/WeatherRunner/service/RunService.java

```diff
@@ -0,0 +1,18 @@
+package UC.WeatherRunner.service;
+
+import java.util.List;
+
+import UC.WeatherRunner.entity.RunEntity;
+
+public interface RunService {
```

Comment:
using the entity we just created, we can now create a service, this holds functions that will be created in the serviceimpl